integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -928,6 +928,99 @@ public void testDomainServerPodRestart(String oldPropertyString, String newPrope
     logger.info("Done - testDomainServerPodRestart");
   }
 
+  /**
+   * Get runtime server yaml file and verify the changed property is in that file.
+   *
+   * @param changedProperty - the changed/added property
+   * @param serverName - server name that is being tested
+   * @throws Exception - test FAILURE Exception if the changed property is not found in the server
+   *     yaml file
+   */
+  public void findServerPropertyChange(String changedProperty, String serverName) throws Exception {
+    logger.info("Inside findServerPropertyChange");
+    // get runtime server pod yaml file
+    String outDir = BaseTest.getUserProjectsDir() + "/weblogic-domains/" + domainUid + "/";
+    StringBuffer command = new StringBuffer();
+    command
+        .append("kubectl get po/")
+        .append(
+            domainUid
+                + "-"
+                + serverName
+                + " -o yaml -n "
+                + domainNS
+                + " > "
+                + outDir
+                + serverName
+                + ".yaml");
+    logger.info("kubectl execut with command: " + command.toString());
+    TestUtils.exec(command.toString());
+
+    try {
+      String content = new String(Files.readAllBytes(Paths.get(outDir + serverName + ".yaml")));
+      boolean result = content.indexOf(changedProperty) >= 0;
+      logger.info(
+          "Server name: "
+              + serverName
+              + " Search result for "
+              + changedProperty
+              + " is: "
+              + result);
+      if (!result) {
+        throw new Exception(
+            "FAILURE: didn't find the property: "
+                + changedProperty
+                + "in the "
+                + serverName
+                + ".yaml file");
+      }
+    } finally {
+      String cmdString = "rm -rf " + outDir + serverName + ".yaml";
+      TestUtils.exec(cmdString);
+    }
+    logger.info("Done - findServerPropertyChange");
+  }
+
+  /**
+   * verify domain server pods get restarted after the property change by kubectl apply -f new
+   * domain yaml file with added/changed property
+   *
+   * @param fileNameWithChangedProperty - the fragment of domain yaml file with new added property
+   *     change
+   * @throws Exception - IOException when file is copied or errors occurred if the tested server is
+   *     not restarted
+   */
+  public void testDomainServerPodRestart(String fileNameWithChangedProperty) throws Exception {
```

Comment:
change the method name to verifyDomainServerPodRestart or some such....we want to keep the test methods in IT classes or BaseTest.java

Reply:
Done